pbs_light.go

```diff
@@ -184,12 +184,10 @@ func getAccountMetrics(id string) *AccountMetrics {
 }
 
 type cookieSyncRequest struct {
-	UUID    string   `json:"uuid"`
```

Comment:
extra fields are ignored right? in other words, legacy requests with UUID continue to work

Reply:
That is correct